internal/pkg/state/view.go

```diff
@@ -180,7 +180,7 @@ func (v *View) loadPowerClaim(ctx context.Context, powerState *power.State, mine
 		return nil, err
 	}
 	if !found {
-		return nil, errors.Errorf("no registered miner %v", miner)
+		return nil, types.ErrNotFound
```

Comment:
This is an unfortunate loss of information. Does anything actually catch and check this error? If not, consider reverting.